splunk_add_on_ucc_framework/utils.py

```diff
@@ -104,3 +106,88 @@ def get_version_from_git() -> str:
     except ValueError:
         raise exceptions.CouldNotVersionFromGitException()
     return f"{version.base}{stage}{version.commit}"
+
+
+def merge_conf_file(
+    src_file: str, dst_file: str, merge_mode: str = "stanza_overwrite"
+) -> None:
+    merge_deny_list = ["default.meta", "README.txt"]
+    if not isfile(src_file):
+        return
```

Comment:
When returning earlier, maybe its worth to log the error ? Or maybe just raise instead of return?

Reply:
Yes, you are right, I have removed this redundant checks as only this current file is using this function.